helical/models/geneformer/geneformer_utils.py

```diff
@@ -258,4 +265,134 @@ def mean_nonpadding_embs(embs, original_lens, dim=1):
     elif embs.dim() == 2:
         masked_embs = embs.masked_fill(~mask, 0.0)
         mean_embs = masked_embs.sum(dim) / original_lens.float()
-    return mean_embs
\ No newline at end of file
+    return mean_embs
+
+# fine-tune Geneformer for classification tasks
+def fine_tuning(
```

Comment:
We use this function within the Model definition. Instead of using two nested functions, I would suggest putting this function into the model definition.

Reply:
I have removed the nested functions and it is now in the fine-tune class definition for Geneformer